contribute.py

```diff
@@ -94,11 +94,11 @@ def arguments():
                         to the repository. The link is accepted in SSH or HTTPS
                         format. For example: git@github.com:user/repo.git or
                         https://github.com/user/repo.git""")
-    parser.add_argument('-un', '--user-name', dest='user_name', type=str, 
-                        required=False, help="""An optional user.name 
+    parser.add_argument('-un', '--user-name', dest='user_name', type=str,
```

Comment:
It would be better to stick to the same naming conventions and use either underscores (`_`) or dashes (`-`) in flags consisting of two words. Could you please update the new flags to use underscores or update the rest of the flags to use dashes if you prefer dashes (in this case readme has to be adjusted as well).